src/parser.ts

```diff
@@ -114,8 +124,8 @@ export function visit(node: unknown, visitor: ASTVisitor, nodeParent?: ASTNode):
   let cont = true
 
   if (visitor[node.type] !== undefined) {
-    // TODO can we avoid this `as any`
-    cont = visitor[node.type]!(node as any, nodeParent)
+    // TODO can we avoid this `as never`
+    cont = visitor[node.type]!(node as never, nodeParent)
```

Comment:
`typescript` now says that this value defaults to `never` I kept the comment as it's probably a very specific typing error that  changed as `typescript` got smarter with new versions